From a2f0e1fd784016336cfbccf6215171115d0bb9e5 Mon Sep 17 00:00:00 2001
From: Henri Verbeet <hverbeet@locutus.nl>
Date: Sat, 6 Dec 2025 20:15:23 +0100
Subject: [PATCH 30/50] wined3d: Get rid of the "restore_pf" field from struct
 wined3d_context_gl.

Its usage was removed in commit ce44dd0ba41b88f6b8eb626863c3358b75aca66c.

Signed-off-by: sewn <sewn@disroot.org>
---
 dlls/wined3d/wined3d_gl.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dlls/wined3d/wined3d_gl.h b/dlls/wined3d/wined3d_gl.h
index c37b329..c18b3fa 100644
--- a/dlls/wined3d/wined3d_gl.h
+++ b/dlls/wined3d/wined3d_gl.h
@@ -644,7 +644,6 @@ struct wined3d_context_gl
     unsigned int level;
     HGLRC restore_ctx;
     HDC restore_dc;
-    int restore_pf;
     HGLRC gl_ctx;
     HDC dc;
     int pixel_format;
-- 
2.52.0

